tools/tpm2_import.c

```diff
@@ -83,6 +84,10 @@ struct tpm_import_ctx {
     TPM2B_DATA enc_sensitive_key;
     TPM2B_MAX_BUFFER encrypted_inner_integrity;
     TPM2B_MAX_BUFFER encrypted_duplicate_sensitive;
+    UINT32 objectAttributes;
+    struct {
+        UINT16 A : 1;
+    } flags;
```

Comment:
I don't think this flag is needed either, you can just check if ctx.objectAttributes != 0

Reply:
This design refers to tpm2_create.c. Yes, if we have -A option only, it is not necessary to use  flags.